MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -910,7 +910,7 @@ private void BuildStreamVideoItem(StreamInfo playlistItem, VideoOptions options,
 
                 // Make sure the video bitrate is lower than bitrate settings but at least 64k
                 var currentValue = playlistItem.VideoBitrate ?? availableBitrateForVideo;
-                playlistItem.VideoBitrate = Math.Clamp(currentValue, 64_000, availableBitrateForVideo);
+                playlistItem.VideoBitrate = Math.Max(Math.Min(availableBitrateForVideo, currentValue), 64_000);
```

Comment:
I'd add a comment that we can't use `Math.Clamp` here bc `availableBitrateForVideo` can be lower then `64_000`
Otherwise I'll run into this again in the future :p